Shared/Components/ImageView.swift

```diff
@@ -47,7 +47,7 @@ struct ImageView: View {
 
     var body: some View {
         if let currentSource = sources.first {
-            LazyImage(url: currentSource.url, transaction: .init(animation: .linear)) { state in
```

Comment:
This does seem to fix the slide in issue, but this removes the nice fade transition between the placeholder (blur hash) and final image. Another solution needs to be figured out with this removed to get the transition _or_ figure out the sliding elsewhere.

My assumption is the issue is in the `BackgroundParallaxHeaderModifier` with how the header is being placed (which is the image) and if there are size differences between the blur hash view and the final image view.